lit.config.json

```diff
@@ -40,5 +40,127 @@
     "derivedVia": "web3.eth.personal.sign",
     "signedMessage": "localhost wants you to sign in with your Ethereum account:\n0xeF71c2604f17Ec6Fc13409DF24EfdC440D240d37\n\nTESTING TESTING 123\n\nURI: https://localhost/login\nVersion: 1\nChain ID: 1\nNonce: gOv4cHte0F8Bg59Mb\nIssued At: 2023-11-17T15:04:32.857Z\nExpiration Time: 2215-07-14T15:04:32.857Z",
     "address": "0xeF71c2604f17Ec6Fc13409DF24EfdC440D240d37"
+  },
+  "CACHE": {
+    "wallet": {
+      "_isSigner": true,
+      "address": "0xeF71c2604f17Ec6Fc13409DF24EfdC440D240d37",
+      "provider": {
```

Comment:
can we remove `wallet` from cache persistence? since we load it from a private key on each run we dont need to save the object here.